packages/create-app/package.json

```diff
@@ -85,6 +85,7 @@
     "@backstage/plugin-scaffolder": "",
     "@backstage/plugin-scaffolder-backend": "",
     "@backstage/plugin-search": "",
+    "@backstage/plugin-search-react": "",
```

Comment:
Oh right!  Will want a changeset for `@backstage/create-app` too.

Reply:
added here https://github.com/backstage/backstage/pull/10842/commits/520e21aaeacc2fba97831071eec1e60ba5722e8f